api/pkg/service/catalog/testdata/TestCatalogError.golden

```diff
@@ -0,0 +1 @@
+display name is missing for buildah task
```

Comment:
you can print the whole api response returned

Reply:
write a http test similar to https://github.com/tektoncd/hub/blob/6c97d9e8d2fda05a60d294e2759978c87dc57594/api/pkg/service/resource/resource_http_test.go#L239